db/schema.rb

```diff
@@ -20,16 +20,16 @@
     t.string   "name",              null: false
     t.boolean  "is_paid"
     t.boolean  "is_permanent_paid"
-    t.datetime "created_at"
-    t.datetime "updated_at"
+    t.datetime "created_at",        null: false
```

Comment:
augh WHAT is this weird schema thing that keeps happening? Obviously something (recent version of rake, maybe?) wants to make these changes. Wish I understood what :(